spec/pool_spec.cr

```diff
@@ -21,7 +21,12 @@ describe DB::Pool do
       fibers.times { channel.receive }
 
       # all numbers were inserted
-      s = fibers * max_n * (max_n + 1) / 2
+      s : Int32
+      {% if compare_versions(Crystal::VERSION, "0.28.0") >= 0 %}
+        s = fibers * max_n * (max_n + 1) // 2
+      {% else %}
+        s = fibers * max_n * (max_n + 1) / 2
```

Comment:
Why the version check? The `//` operator already exists in crystal 0.27.

Reply:
crystal-db itself works still in 0.25. When that change I will remove these checks.

Yeah, 0.25 seems old. I don’t really expect to be used but there is no need to upgrade crystal-db to requiere newer Crystal.